client/my-sites/checkout/composite-checkout/components/refund-policies.tsx

```diff
@@ -33,7 +37,27 @@ export enum RefundPolicy {
 }
 
 export function getRefundPolicies( cart: ResponseCart ): RefundPolicy[] {
+	const isGiftPurchase = cart.is_gift_purchase;
+
 	const refundPolicies: Array< RefundPolicy | undefined > = cart.products.map( ( product ) => {
+		if ( isGiftPurchase ) {
+			if ( isDomainRegistration( product ) ) {
+				return RefundPolicy.GiftDomainPurchase;
+			}
+
+			if ( isMonthlyProduct( product ) ) {
+				return RefundPolicy.GiftMonthlyPurchase;
+			}
+
+			if ( isYearly( product ) ) {
+				return RefundPolicy.GiftYearlyPurchase;
+			}
+
+			if ( isBiennially( product ) ) {
+				return RefundPolicy.GiftBiennialPurchase;
+			}
```

Comment:
We can probably simplify this to not check for gift term, since you can only gift a domain and/or a plan.

```suggestion
			return RefundPolicy.GiftPlanPurchase;
```

Reply:
You can ignore my comment: p1669821415337469/1669748340.840929-slack-C0117V2PCAE